1_Record_dimension.R

```diff
@@ -1,61 +1,92 @@
 
 library(sf)
-library(maptools)
+library(terra)
 library(spatstat)
-library(raster)
+library(geodata)
+library(dplyr)
+library(vroom)
 
 # ---------------------------
 #   Data loading
 # ---------------------------
-wd <- 'F:/GSI/'
-setwd(wd)
-dir.create("1_Records/")
 
-col<-getData(name = 'GADM', country = 'COL', level = 0) #Areo of interes (AOI)
-col@data$OBJECTID<-1
+# Definición del directorio de trabajo
```

Comment:
Comments must be in English to be included in the changes. Please replace it with that language in all of the comments.